src/pages/Transaction/TransactionCellScript/index.tsx

```diff
@@ -333,8 +331,42 @@ export default ({ cell, onClose, txStatus }: { cell: State.Cell; onClose: Functi
           <div className="transaction__detail_copy">
             <TransactionDetailCopyButton onClick={onClickCopy}>
               <div>{i18n.t('common.copy')}</div>
-              <img src={isMainnet() ? CopyIcon : CopyBlueIcon} alt="copy" />
+              <svg width="24" height="24" viewBox="0 0 24 24" fill="none">
+                <path
+                  fillRule="evenodd"
+                  clipRule="evenodd"
+                  d="M9 2H20C21.1046 2 22 2.89543 22 4V16C22 17.1046 21.1046 18 20 18H19V8C19 5.79086 17.2091 4 15 4L7 4C7 2.89543 7.89543 2 9 2ZM9 0C6.79086 0 5 1.79086 5 4L4 4C1.79086 4 0 5.79086 0 8V20C0 22.2091 1.79086 24 4 24H15C17.2091 24 19 22.2091 19 20H20C22.2091 20 24 18.2091 24 16V4C24 1.79086 22.2091 0 20 0H9ZM15 6H4C2.89543 6 2 6.89543 2 8V20C2 21.1046 2.89543 22 4 22H15C16.1046 22 17 21.1046 17 20V8C17 6.89543 16.1046 6 15 6ZM6 8C5.44772 8 5 8.44771 5 9C5 9.55229 5.44772 10 6 10H10C10.5523 10 11 9.55229 11 9C11 8.44771 10.5523 8 10 8H6ZM5 18C5 17.4477 5.44772 17 6 17H10C10.5523 17 11 17.4477 11 18C11 18.5523 10.5523 19 10 19H6C5.44772 19 5 18.5523 5 18ZM6 12C5.44772 12 5 12.4477 5 13C5 13.5523 5.44772 14 6 14H13C13.5523 14 14 13.5523 14 13C14 12.4477 13.5523 12 13 12H6Z"
+                  fill="none"
+                />
+              </svg>
```

Comment:
SVG could be imported as an independent component so it could be reviewed by image viewer, and makes the code clean

Reply:
> SVG could be imported as an independent component so it could be reviewed by image viewer, and makes the code clean

moved svg icons into separate files